solutions/usaco-831.mdx

```diff
@@ -2,13 +2,124 @@
 id: usaco-831
 source: USACO Bronze 2018 Open
 title: Team Tic Tac Toe
-author: Jesse Choe, Kevin Sheng, Benjamin Qi
+author: Jesse Choe, Kevin Sheng, Benjamin Qi, Brad Ma
 ---
 
 ## Solution 1
 
 [Official Analysis](http://www.usaco.org/current/data/sol_tttt_bronze_open18.html)
 
+<LanguageSection>
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class TeamTicTacToe {
+	static Character[][] B = new Character[3][3];
+
+	// Does 1 cow win?
+	static int cowWins(char ch) {
+		// Check diagonals
+		if (B[0][0] == ch && B[1][1] == ch && B[2][2] == ch) {
+			return 1;
+		}
+		if (B[0][2] == ch && B[1][1] == ch && B[2][0] == ch) {
+			return 1;
+		}
+
+		// Check rows and columns
+		for (int i=0; i<3; i++) {
```

Comment:
can you be consistent with the spacing between ops at least

Reply:
ik i copied parts of the official code and I forgot to do this

Reply:
oops